Source/Core/DolphinQt/Resources.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2015 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QList>
+#include <QPixmap>
+
+// Store for various QPixmaps that will be used repeatedly.
+class Resources final
```

Comment:
The constructor for this should be private if it's a utility class (or be made into a namespace, so the internals aren't part of its interface).